tests/unit/utils/files.spec.jsx

```diff
@@ -0,0 +1,112 @@
+import { expect } from 'vitest'
+import { filesValidation } from '~/utils/validations/files'
+
+const mockedValidation = {
+  maxFileSize: 5_000_000,
+  maxAllFilesSize: 20_000_000,
+  filesTypes: ['application/pdf', 'image/jpeg', 'image/png'],
+  fileSizeError: 'common.fileSizeError',
+  allFilesSizeError: 'common.allFilesSizeError',
+  typeError: 'common.typeError',
+  maxQuantityFiles: 7,
+  maxFileNameLength: 55,
+  maxFileNameError: 'common.fileNameError'
+}
+
+const tooLongNameFile = [
+  {
+    lastModified: 1676387080931,
+    lastModifiedDate: new Date(),
+    name: 'wwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww.jpg',
+    size: 26715,
+    type: 'image/jpeg',
+    webkitRelativePath: ''
+  }
+]
+
+const tooBigFile = [
+  {
+    lastModified: 1676387080931,
+    lastModifiedDate: new Date(),
+    name: 'wwwwwwwwwwwwwwwww.jpg',
+    size: 5_000_002,
+    type: 'image/jpeg',
+    webkitRelativePath: ''
+  }
+]
+
+const wrongTypeFile = [
+  {
+    lastModified: 1676387080931,
+    lastModifiedDate: new Date(),
+    name: 'wwwwwwwwwwwwwwwww.jpg',
+    size: 5_000_000,
+    type: 'image/bmp',
+    webkitRelativePath: ''
+  }
+]
+
+const overallSizeTooBig = [
+  {
+    lastModified: 1676387080931,
+    lastModifiedDate: new Date(),
+    name: 'wwwwwwwwwwwwwwwww.jpg',
+    size: 5_000_000,
+    type: 'image/jpeg',
+    webkitRelativePath: ''
+  },
+  {
+    lastModified: 1676387080931,
+    lastModifiedDate: new Date(),
+    name: 'wwwwwwwwwwwwwwwww.jpg',
+    size: 5_000_000,
+    type: 'image/jpeg',
+    webkitRelativePath: ''
+  },
+  {
+    lastModified: 1676387080931,
+    lastModifiedDate: new Date(),
+    name: 'wwwwwwwwwwwwwwwww.jpg',
+    size: 5_000_000,
+    type: 'image/jpeg',
+    webkitRelativePath: ''
+  },
+  {
+    lastModified: 1676387080931,
+    lastModifiedDate: new Date(),
+    name: 'wwwwwwwwwwwwwwwww.jpg',
+    size: 5_000_000,
+    type: 'image/jpeg',
+    webkitRelativePath: ''
+  },
+  {
+    lastModified: 1676387080931,
+    lastModifiedDate: new Date(),
+    name: 'wwwwwwwwwwwwwwwww.jpg',
+    size: 5_000_000,
+    type: 'image/jpeg',
+    webkitRelativePath: ''
```

Comment:
instead of repeating same fields with value for each file, you can create an initial file and then use spread operator